packages/devtools/lib/src/inspector/inspector.css

```diff
@@ -134,3 +134,16 @@ messages beautiful.
 .inspector-level-off {
 
 }
+
+.expand-collapse-container {
+    position: sticky;
```

Comment:
Can this be absolute instead as we aren't really leveraging any sticky behavior?

Reply:
that is absolute position. Sticky does a bit more than that. https://www.w3schools.com/howto/howto_css_sticky_element.asp

Reply:
If i give this absolute position and it's parent relative position, the button group still moves around as you scroll around the tree.

Reply:
you probably need to attach the absolute position element to the parent node of the element it is being attached to.